src/openrct2/world/Banner.h

```diff
@@ -42,6 +43,7 @@ struct Banner
     }
 
     std::string GetText() const;
+    size_t FormatTextTo(void* args, bool add_colour) const;
```

Comment:
We use camelCase for variables

Reply:
Oh great! That's what I'm actually used to :)